### PR TITLE
Add more instructions and timings

### PR DESCRIPTION
**Pull request by rodrigonogueira4**
- Some instructions to facilitate installation.
- More timings for GPT-NeoX-20B.
- Add command to start training Galactica 30B on a v2-256.
- Remove GPT-J cosine scheduler as it does not work well with Adafactor.
- Small changes from the original T5x repo.

### CODE REVIEW

**Review by piresramon — APPROVED** [review]

File: README_TPU_VM_POD.md [hunk 1]
```diff
@@ -166,7 +177,7 @@ gcloud alpha compute tpus tpu-vm ssh $TPU_NAME \
 
 gcloud alpha compute tpus tpu-vm ssh $TPU_NAME \
   --zone $ZONE \
-  --worker=all \
+  --worker=0,1 \
```
Comment (hugoabonizio) [inline README_TPU_VM_POD.md:180]: Why not `all` in this case?

Comment (rodrigonogueira4) [reply README_TPU_VM_POD.md:180]: I added a comment above explaning the reason. Makes sense?

File: t5x/contrib/moe/trainer_test.py [hunk 2]
```diff
@@ -34,7 +34,7 @@
 
 # Make `log_elapsed_time` a no-op to simplify mocking of `time.time()`.
 @contextlib.contextmanager
-def fake_log_elapsed_time(_):
+def fake_log_elapsed_time(_, event=None):  # pylint: disable=unused-argument
```
Comment (ZanezZephyrs) [inline t5x/contrib/moe/trainer_test.py:37]: why was that necessary?

File: t5x/examples/decoder_only/examples/base_mc4_from_scratch.gin [hunk 3]
```diff
@@ -64,15 +64,4 @@ utils.SaveCheckpointConfig:
 models.DecoderOnlyModel.predict_batch_with_aux.num_decodes = 8
 models.DecoderOnlyModel.inputs_bidirectional_attention = False
 
-partitioning.PjitPartitioner.num_partitions = 2
-
-trainer.Trainer:
-  num_microbatches = None
-  learning_rate_fn = @optax.warmup_cosine_decay_schedule()
-
-optax.warmup_cosine_decay_schedule:
```
Comment (ZanezZephyrs) [inline t5x/examples/decoder_only/examples/base_mc4_from_scratch.gin:73]: where is the scheduler being defined now?

**Review by ZanezZephyrs — APPROVED** [review]

File: t5x/examples/decoder_only/examples/base_mc4_from_scratch.gin (hunk 3 above)
Comment (rodrigonogueira4) [reply t5x/examples/decoder_only/examples/base_mc4_from_scratch.gin:73]: here: https://github.com/rodrigonogueira4/t5x/blob/cb28d2a41dbc14a31a992af26404739a969224b0/t5x/configs/runs/pretrain.gin#L105

File: t5x/contrib/moe/trainer_test.py (hunk 2 above)
Comment (rodrigonogueira4) [reply t5x/contrib/moe/trainer_test.py:37]: It was a fork sync from the original Google's repo.
